public/static/locales/el/common.json

```
@@ -0,0 +1,19 @@
+{
+	"slogan": "H πρώτη Ελληνική λέσχη πληροφορικής για εφήβους",
+	"titlePrefix": "Είμαστε η",
+	"articleQuestion": "Σε ενδιαφέρει η πληροφορική;",
+	"description": "Η Hackropolis είναι μία διαδικτυακή λέσχη πληροφορικής για ελληνόφωνους εφήβους που ενδιαφέρονται για την πληροφορική και άλλους σχετικούς τομείς. Οργανώνουμε συχνά διαδικτυακές συναντήσεις και φτιάχνουμε μαζί εφαρμογές και προγράμματα. Δεν απαιτείται προυπάρχουσα εμπειρία - το μονο που χρειάζεσαι είναι σύνδεση στο διαδίκτυο!",
+	"operateUnder": "Αποτελούμαστε κυρίως από μαθητές δευτεροβάθμιας και τριτοβάθμιας εκπαίδευσης και λειτουργούμε υπό την αιγίδα του",
+	"nonProfit": "(εγγεγραμένη 501(c)(3) μη κερδοσκοπική οργάνωση στην Καλιφόρνια, ΗΠΑ), που υποστηρίζει παρόμοιες λέσχες σαν την δική μας ανά τον κόσμο",
```

no need for the 501 stuff since its all a MKO under greek law (i should probably try and get ourselves recognised in grc)
```suggestion
	"nonProfit": "(μιας μη κερδοσκοπική οργάνωση στην Καλιφόρνια των ΗΠΑ), που υποστηρίζει παρόμοιες λέσχες σαν την δική μας ανά τον κόσμο",
```